test/common/parse-util_test.c

```diff
@@ -29,11 +30,40 @@ void test_parse_long(const char *input, int expectedReturn, long expectedResult)
         }
 }
 
+void test_parse_port(const char *input, bool expected_return, uint16_t expected_result) {
+        char *msg = NULL;
+
+        uint16_t res = 0;
+        bool ret = parse_port(input, &res);
+        if ((ret != expected_return) || (res != expected_result)) {
+                asprintf(&msg,
+                         "FAILED: %s\n\tInput: '%s'\n\tExpected: Return=%d, Result=%ld\n\tGot: Return=%d, Result=%ld\n",
```

Comment:
```suggestion
                         "FAILED: %s\n\tInput: '%s'\n\tExpected: Return=%d, Result=%d\n\tGot: Return=%d, Result=%d\n",
```
The result for `parse_port` is an `int`, so `%d` instead of `%ld` should be used.

Reply:
Done